bin/dredd

```diff
@@ -33,18 +33,38 @@ if (argv._[1] === undefined) {
   console.error("Error: Must specify api endpoint.");
   argError = true;
 }
+
 if (argError) {
   console.error("\n");
   optimist.showHelp(fn=console.error);
   process.exit(1);
 }
 
+// transform path and p argument to array if it's not
+if(!Array.isArray(argv['path'])){
```

Comment:
This will work only for `only` arg, because some terminals are expanding globs and concating them as CLI arguments and I need [fill](https://github.com/apiaryio/dredd/pull/125/files#diff-1250f065cb350c09960352445f4d42ddR53) those into `option['path']` which couldn't be an array at that moment.